models/onepunchswap/bnb/onepunchswap_bnb_trades.sql

```diff
@@ -0,0 +1,126 @@
+    post_hook='{{ expose_spells(\'["bnb"]\',
+                                "project",
+                                "onepunchswap",
+                                \'["onepunchteam"]\') }}'
+    )
+}}
+
+{% set project_start_date = '2023-01-01' %}
+{% set hashflow_bnb_evt_trade_tables = [
+    source('onepunch_normal_bnb', 'DexLiquidityProvider_evt_QuoteAccepted')
+    , source('onepunch_quick_bnb', 'QuickLiquidityProvider_evt_QuoteAccepted')
+] %}
+
+WITH dexs AS(
+  {% for evt_trade_table in hashflow_bnb_evt_trade_tables %}
+    SELECT
+        evt_block_time                AS block_time,
+        user                          AS taker,
+        ''                              AS maker,
+        cast(get_json_object(quoteInfo,'$.toAmount') as double) as token_bought_amount_raw,
+        cast(get_json_object(quoteInfo,'$.fromAmount') as double) as token_sold_amount_raw,
+        cast(NULL as double)            AS amount_usd,
+        case 
+            when get_json_object(quoteInfo,'$.toAsset') = '0x0000000000000000000000000000000000000000' then '0xbb4cdb9cbd36b01bd1cbaebf2de08d9173bc095c'
+            else get_json_object(quoteInfo,'$.toAsset')
+        end                                                 as token_bought_address,
+        case 
+            when get_json_object(quoteInfo,'$.fromAsset') = '0x0000000000000000000000000000000000000000' then '0xbb4cdb9cbd36b01bd1cbaebf2de08d9173bc095c'
+            else get_json_object(quoteInfo,'$.fromAsset')
+        end                                                 as token_sold_address,
+
+        contract_address              AS project_contract_address,
+        evt_tx_hash                   AS tx_hash,
+        ''                              AS trace_address,
+        evt_index
+      FROM
+        {{ evt_trade_table }}
+        {% if not is_incremental() %}
+        WHERE evt_block_time >= '{{project_start_date}}'
+        {% endif %}
+        {% if is_incremental() %}
+        WHERE evt_block_time >= date_trunc("day", now() - interval '1 week')
+        {% endif %}
+
+        {% if not loop.last %}
+        UNION ALL
+        {% endif %}
+
+    {% endfor %}
+)
+
+SELECT
+    'bnb'                                                        AS blockchain
+    , 'onepunchswap'                                             AS project
+    , case
+        when dexs.project_contract_address = '0xeeb28c597dc67ed4a337c14b20b0a5c353e38253' then 'quick'
+        else 'normal'
+      end                                                        AS version           
+    , TRY_CAST(date_trunc('DAY', dexs.block_time) AS date)       AS block_date
+    , dexs.block_time
+    , bep20a.symbol                                              AS token_bought_symbol
+    , bep20b.symbol                                              AS token_sold_symbol
+    , dexs.token_bought_address                                  AS token_bought_address
+    , case
```

Comment:
`token_sold_address` may be missed here.
```suggestion
    , dexs.token_bought_address                                  AS token_bought_address
    , dexs.token_sold_address                                  AS token_sold_address
    , case
```